superset/utils/webdriver.py

```diff
@@ -141,6 +141,7 @@ def get_screenshot(
             img = element.screenshot_as_png
         except TimeoutException:
             logger.warning("Selenium timed out requesting url %s", url, exc_info=True)
+            img = element.screenshot_as_png
```

Comment:
is it always possible to call `element.screenshot_as_png` without raising when a `TimeoutException` occurs?

Reply:
i guess it's webdriver raise TimeoutException right? i am not sure how to not raise it...